### PR TITLE
Line layer pop up shows values

### PR DESCRIPTION
**Pull request by aeischeid**
render the features entry properties with just very basic styling

image

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -65,8 +61,13 @@
 			sourceLayer="rnet_limerick"
 			hoverCursor="pointer"
 		>
-			<Popup openOn="click" offset={[0, -10]}>
-				<div class="text-lg font-bold">"Test value"</div>
+			<Popup openOn="click" offset={[0, -10]} let:features>
+				{@const props = features?.[0]?.properties}
```
Comment (Robinlovelace) [inline src/routes/+page.svelte:65]: This is the magic I need to get my head around. One line of code, lots to digest!

File: src/routes/+page.svelte [hunk 2]
```diff
@@ -37,11 +34,10 @@
 		}}
 	/>
 
-    <!-- <Control position="top-right"/> -->
-    <NavigationControl position="top-right"/>
-    <GeolocateControl position="top-right"/>
-    <FullscreenControl position="top-right"/>
-
+	<!-- <Control position="top-right"/> -->
```
Comment (Robinlovelace) [inline src/routes/+page.svelte:37]: Out of interest how to auto format/indent?

File: src/routes/+page.svelte [hunk 3]
```diff
@@ -23,7 +20,7 @@
 	center={[-8.63, 52.66]}
 	zoom={11}
 	class="map"
-    controlPosition="top-right"
+	controlPosition="top-right"
```
Comment (Robinlovelace) [inline src/routes/+page.svelte:23]: :+1:

**Review by Robinlovelace — APPROVED** [review]

File: src/routes/+page.svelte (hunk 2 above)
Comment (aeischeid) [reply src/routes/+page.svelte:37]: in vs code, can use `ctrl + shift + i` to trigger the prettier formatter. can always run prettier from CLI too.

File: src/routes/+page.svelte (hunk 1 above)
Comment (aeischeid) [reply src/routes/+page.svelte:65]: not sure my summary will be fully correct, but can try;  The component exposes some of it's state. to hook into that use `let:[theVarNameExposed]` now that is available for you to use within the children of the component that you define up here.

File: src/routes/+page.svelte (hunk 2 above)
Comment (Robinlovelace) [reply src/routes/+page.svelte:37]: Makes sense, and I use VS Code.

**Pull request merged**